modules/sdk-coin-avaxp/src/lib/utils.ts

```diff
@@ -37,7 +37,7 @@ export class Utils implements BaseUtils {
    */
   /** @inheritdoc */
   isValidAddress(address: string): boolean {
-    return (address as string).match(/^[a-zA-Z0-9]+$/) !== null;
+    return /^P-[a-zA-Z0-9]+$/.test(address);
```

Comment:
https://github.com/BitGo/bitgo-microservices/pull/21198#discussion_r927758537

Reply:
and also, it's not always start with `P-` cuz node uses it, can we do something like 
return `/^P-[a-zA-Z0-9]+$/.test(address); || /^NodeId-[a-zA-Z0-9]+$/.test(address)`

Reply:
fixed